src/components/Garden/DecisionDetail/VoteStatus.tsx

```diff
@@ -5,7 +5,35 @@ import { IconCheck, IconCross, GU, textStyle, useTheme } from '@1hive/1hive-ui'
 import celesteIconSvg from '@assets/celeste-icon.svg'
 import challengeIconSvg from '@assets/challenge-icon.svg'
 
-export const getStatusAttributes = (vote, theme) => {
+type AttributesProps = {
+  label: string
+  Icon?: React.ReactNode
+  iconSrc?: any
+  color: string
+  background?: string
+  borderColor?: string
+}
+
+type VoteType = {
+  isAccepted: boolean
+  isDelayed: boolean
+  statusData: {
+    open: boolean
+    cancelled: boolean
+    rejected: boolean
+    accepted: boolean
+    pendingExecution: boolean
+    executed: boolean
+    challenged: boolean
+    settled: boolean
+    disputed: boolean
+  }
+}
+
+export const getStatusAttributes = (
+  vote: any,
+  theme: any
+): AttributesProps | undefined => {
```

Comment:
The name Props its used to arguments, right? If it type of return maybe should be called StatusAttributeType or something like that?

Reply:
It could yes. But when is a group of types as props I used the "Props" in the name, since the Type is implicit.